nativeHelper.gradle.kts

```diff
@@ -0,0 +1,11 @@
+val hostOs = System.getProperty("os.name")
```

Comment:
You can replace all of this with the `HostManager` class, which comes with the kotlin gradle plugin. You can then do stuff like, `HostManager.isMingw` and `HostManager.isLinux`.

Reply:
Changed over thanks